File: src/CSharp/SimpleOperations/RandomSequence.cs
using System;
using System.Diagnostics;
using Kitware.VTK;

void RandomSequence() {
   //Create a random sequence generator.
   vtkMinimalStandardRandomSequence sequence = 
      vtkMinimalStandardRandomSequence.New();

   // initialize the sequence
   sequence.SetSeed((int)(DateTime.Now.Ticks & 0x0000FFFF));
   //Get 3 random numbers.
   double x = sequence.GetValue();
   sequence.Next();
   double y = sequence.GetValue();
   sequence.Next();
   double z = sequence.GetValue();

   // You can also use sequence.GetRangeValue(-1.0, 1.0); 
   // to set a range on the random values.

   // Output the resulting random numbers
   Console.WriteLine("x: " + x + " y: " + y + " z: " + z);
}

File: src/Java/SimpleOperations/RandomSequence.java
import vtk.vtkNativeLibrary;
import vtk.vtkMinimalStandardRandomSequence;


public class RandomSequence 
{
  // -----------------------------------------------------------------
  // Load VTK library and print which library was not properly loaded
  static 
  {
    if (!vtkNativeLibrary.LoadAllNativeLibraries()) 
    {
      for (vtkNativeLibrary lib : vtkNativeLibrary.values()) 
      {
        if (!lib.IsLoaded()) 
        {
          System.out.println(lib.GetLibraryName() + " not loaded");
        }
      }
    }
    vtkNativeLibrary.DisableOutputWindow(null);
  }
  // -----------------------------------------------------------------
	
  public static void main(String s[]) 
  {					  
    //Create a random sequence generator.
    vtkMinimalStandardRandomSequence Sequence = new vtkMinimalStandardRandomSequence();
    //initialize the sequence
    Sequence.SetSeed(1);

    //Get 3 random numbers.
    double x = Sequence.GetValue();
    Sequence.Next();
    double y = Sequence.GetValue();
    Sequence.Next();
    double z = Sequence.GetValue();
    Sequence.Next();

    // You can also use Sequence.GetRangeValue(-1.0, 1.0) to set a range on the random values.

    System.out.println("X:"+x + "\n" + "Y:" + y + "\n" + "Z:"+ z);
    
  }		  
}
